app/src/main/java/com/nervousfish/nervousfish/data_objects/Contact.java

```diff
@@ -35,16 +35,10 @@ public Contact(final String name, final Collection<IKey> keys) {
         this.keys.addAll(keys);
     }
 
-    /**
-     * Get the name of the {@link Contact}.
-     */
     public String getName() {
         return this.name;
     }
 
-    /**
-     * Get the public key of the {@link Contact}.
```

Comment:
according to Stackoverflow it's not needed, the functionality of a getter should be clear from the name. Checkstyle won't give a warning if it's a getter or a setter